lua/menu.lua

```diff
@@ -1012,12 +1012,13 @@ end
 -- DEVICES
 m.devices = {}
 m.devices.pos = 1
-m.devices.list = {"midi", "grid"}
+m.devices.list = {"midi", "grid", "arc"}
 m.devices.len = #m.devices.list
 function m.devices.refresh()
   m.devices.options = {
     midi = {"none"},
-    grid = {"none"}
+    grid = {"none"},
+    arc = {"none"},
```

Comment:
is the comma here a problem?

Reply:
no, it's just for convenience. won't have to add comma when adding more lines